Release version 1.0.6

Sidsector9 opened the pull request:
Closes #167

Sidsector9 committed:
update version

Sidsector9 committed:
update CHANGELOG.md and readme.txt

Sidsector9 committed:
update CREDITS.md

jeffpaul reviewed (dismissed):
sorry for the slow review, mea culpa!

jeffpaul commented:
Noting that there are additional changes to get added to the changelogs (and may have a couple others in the milestone worth reviewing/merging and similarly adding to the changelogs as well).

Sidsector9 committed:
Merge branch 'develop' into release/1.0.6

Sidsector9 committed:
add build pre-release action

Sidsector9 committed:
update CHANGELOG and readme.txt

Sidsector9 committed:
exclude /bin from deployment

Sidsector9 commented:
@jeffpaul I've updated the following:

- CHANGELOG and readme.txt
- added pre-release ZIP Github action
- resolved conflicts

Can you review? This can be released today.

jeffpaul commented:
Don't worry about the changelog date, we can update that after release next week

jeffpaul commented:
@Sidsector9 there's like a LOT more that's been merged to `develop` since you started this release... want to update to include all those items?

Sidsector9 commented:
I have tracked it, will be focusing on OSP tomorrow 👍

Sidsector9 committed:
Merge branch 'develop' into release/1.0.6

Sidsector9 committed:
update README and credits

Sidsector9 committed:
update release date

Sidsector9 committed:
fix link

cadic committed:
Merge branch 'develop' into release/1.0.6

jeffpaul commented:
@Sidsector9 some merge conflicts here, and likely some additional PRs that have since landed in `develop` that we'll want to account for in the changelogs

Sidsector9 committed:
Merge remote-tracking branch 'origin/develop' into release/1.0.6

Sidsector9 committed:
update CHANGELOG.md and readme.txt

jeffpaul committed:
Update CHANGELOG.md

jeffpaul committed:
Update CREDITS.md

jeffpaul committed:
Update readme.txt

jeffpaul committed:
Update CHANGELOG.md

jeffpaul reviewed (approved):
Looks good, thanks for getting this release out!
